src/devices/Bh1745/Bh1745.cs

```diff
@@ -35,10 +42,11 @@ public class Bh1745 : IDisposable
         public Bh1745(I2cDevice device)
         {
             _i2cDevice = device;
-            ChannelCompensationMultipliers = new ChannelCompensationMultipliers
-            {
-                Red = 2.2, Green = 1.0, Blue = 1.8, Clear = 10.0
-            };
+            ChannelCompensationMultipliers = new (
```

Comment:
While I don't have a big preference, I suppose that object initializer still works with records right? Like:
```c#
ChannelCompensationMultipliers = new ()
            {
                Red = 2.2, Green = 1.0, Blue = 1.8, Clear = 10.0
            };
```

Reply:
If I do that, then the constructor isn't satisfied, and then I cannot use the pretty records syntax.